src/components/ui/ListItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode } from "react";
+import {
+  ListItem as RNListItem,
+  ListItemProps as RNListItemProps,
+} from "react-native-elements";
+import { Icon } from "..";
+import { colors } from "@app/utils";
+
+interface ListItemProps extends RNListItemProps {
+  id: string;
+  title: string;
+  otherInformation: ReactNode | string; //for other elements to display below the title e.g. description and pending bar
+  iconName?: string;
+  iconColor?: string;
+  iconFamily?: string;
+  showNav: boolean; //show chevron right navigation icon
```

Comment:
maybe we could add a default value to have this as true by default since most list item has it